src/components/AddAdditionalRoles.tsx

```diff
@@ -47,29 +62,41 @@ export default function AddAdditionalRolesComponent() {
-                                        <option value="Test Manager">Test Manager</option>
-                                        <option value="Technical writer">Technical writer</option>
-                                        <option value="Key User">Key User</option>
+                                    <select className="form-select" key ="" name="role" value={elem.role} aria-label="additional role" defaultValue = {defaultValue()} onChange={e => handleInputChange(e, j)}>
+                                        {
+                                        documentType === DocumentTypes.COMPONENT
+                                        ?
+                                        <>
+                                            <option value="Committer">Committer</option>
+                                            <option value="Contributor">Contributor</option>
+                                            <option value="Expert">Expert</option>
+                                        </>
+                                        :
+                                        <>
+                                            <option value="Stakeholder">Stakeholder</option>
+                                            <option value="Analyst">Analyst</option>
+                                            <option value="Contributor">Contributor</option>
+                                            <option value="Accountant">Accountant</option>
+                                            <option value="End User">End User</option>
+                                            <option value="Quality Manager">Quality Manager</option>
+                                            <option value="Test Manager">Test Manager</option>
+                                            <option value="Technical writer">Technical writer</option>
+                                            <option value="Key User">Key User</option>
+                                        </>
+                                        }
                                     </select>
                                 </div>
                                 <div className="col-lg-5">
                                     <input name="email" value={elem.email} type="email" onChange={e => handleInputChange(e, j)} className="form-control" placeholder={`Enter email`} aria-describedby={`Email`} />
                                 </div>
                                 <div className="col-lg-2">
-                                    <button type="button" onClick={() => handleRemoveClick(j)} className={`fw-bold btn btn-light ${styles['button-plain']}`}><i className="bi bi-trash3-fill"></i></button>
+                                    <button type="button" onClick={() => handleRemoveClick(j)} className={`fw-bold btn btn-light ${styles['button-plain']}`}><FaTrashAlt className="bi bi-trash3-fill"></FaTrashAlt></button>
```

Comment:
Don'tt add inline styles, use global css if the button is common, or component css for specialized